cypress-tests/cypress/e2e/PaymentUtils/Utils.js

```diff
@@ -39,13 +40,19 @@ const connectorDetails = {
   datatrans: datatransConnectorDetails,
   wellsfargo: wellsfargoConnectorDetails,
   fiuu: fiuuConnectorDetails,
+  worldpay: worldpayConnectorDetails,
 };
 
 export default function getConnectorDetails(connectorId) {
   let x = mergeDetails(connectorId);
   return x;
 }
 
+export function getConnectorFlowDetails(connectorData, commonData, key) {
+  let data = connectorData[key] === undefined ? commonData[key] : connectorData[key];
+  return data;
+}
+
```

Comment:
It does have a very limited scope as of now, we can remove this once we introduce custom deep merge (merge everything except `Response`). But we can let it be for now, since it's out of scope of this PR